backend/bot/index.js

```diff
@@ -14,7 +29,23 @@ bot.on('start', async () => {
   }
 });
 
-const postResultToSlack = (match) => {
+const updateSlack = async (match) => {
+  // Update channel purpose
+  const users = await storage.getAllUsers(true);
+  const purposeCharacterLimit = 250;
+  let newChannelPurpose = '';
+  let i = 1;
+  for(const user of users) {
```

Comment:
learn to program my fren 
if users is an array, then array.forEach(), otherwise
is 
Object.entries(users).forEach()
both look better than having a for loop and a while loop at one place

Reply:
ugggh, ugly!